Session token "safety window" when refreshing (#22503)
directus/directus: docs/self-hosted/config-options.md

| `SESSION_COOKIE_SECURE`             | Whether or not to set the `secure` attribute for the session cookie.                                                                                                                                 | `false`                   |
| `SESSION_COOKIE_SAME_SITE`          | Value for `sameSite` in the session cookie.                                                                                                                                                          | `lax`                     |
| `SESSION_COOKIE_NAME`               | Name of the session cookie.                                                                                                                                                                          | `directus_session_token`  |

> | `SESSION_REFRESH_GRACE_PERIOD`      | The duration where a refresh request will allow recently refreshed sessions to be used gracefully preventing race conditions in refresh calls                                                        | `10s`                     |

Reviewer: ```suggestion
| `SESSION_REFRESH_GRACE_PERIOD`      | The duration during which a refresh request will permit recently refreshed sessions to be used, thereby preventing race conditions in refresh calls                                            | `10s`                     |
```
